content/en/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -705,7 +705,7 @@ Patch the StatefulSet again to change the container image that this
 StatefulSet uses:
 
 ```shell
-kubectl patch statefulset web --type='json' -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/image", "value":"registry.k8s.io/nginx-slim:0.7"}]'
+kubectl patch statefulset web --type='json' -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/image", "value":"registry.k8s.io/nginx-slim:0.24"}]'
```

Comment:
I'm not sure this works. We need two **different** image tags; they can be any two tags, but they can't be the same as each other.